docs/recipes/Running-In-Production.md

```diff
@@ -156,4 +159,64 @@ One App is configured via [environment variables](#). There are a few
 environment variables that are required for One App to start up including the one used to let One App
 know where to fetch a module map from as described above.
 
+### Minimum Required Environment Variables:
+
+In order for One App to start after it has been deployed, the following environment variables must be set before attempting to deploy / start the One App container:
+
+- `HOLOCRON_MODULE_MAP_URL`
+- `ONE_CLIENT_CDN_URL`
+- `ONE_CLIENT_CSP_REPORTING_URL`
+- `ONE_CLIENT_REPORTING_URL`
+- `ONE_CLIENT_ROOT_MODULE_NAME`
+
+
+Example:
+
+```bash
+HOLOCRON_MODULE_MAP_URL="https://example-module-map.vercel.app/"
+ONE_CLIENT_CDN_URL="https://www.my-app-domain.com/_/static/"
+ONE_CLIENT_CSP_REPORTING_URL="https://www.my-app-domain.com/_/report/security/csp-violation"
+ONE_CLIENT_REPORTING_URL="https://www.my-app-domain.com/_/report/errors"
+ONE_CLIENT_ROOT_MODULE_NAME="my-root-module-name"
+```
+
+## Deploying to AWS Elastic Beanstalk
+
+Prerequisites:
+
+- AWS account
+- A valid module map with at least the root module published and deployed. You can follow [this guide](Publishing-Modules.md) to create your module map and deploy your modules.
+
+You can deploy the published [One App production image](https://hub.docker.com/r/oneamex/one-app) from Docker Hub to Elastic Beanstalk by following these steps:
+
+1. Go to the AWS console, select Elastic Beanstalk and click on "Create Application".
+2. Give your application a name and select "Docker" as the platform.
+3. Select "Sample Application" and click "Create Application".
+4. Before we can upload and install the One App image to our new environment, we need to add the [Minimum Required Environment Variables](#minimum-required-environment-variables). Click on the "Configuration" of your new environment,
+under the "Software" category click "Edit". Enter all the required environment variables under the "Environment Properties" section.
+
+> Important: Ensure that you have entered a valid `HOLOCRON_MODULE_MAP_URL` that is publicly accessible and that at least the root module specified in `ONE_CLIENT_ROOT_MODULE_NAME` is published; failure to do so, will result in One App crashing and unable to start.
+
```

Comment:
images would be amazing for the step by step. Maybe something we can consider for the future